mappings/net/minecraft/world/gen/UniformIntDistribution.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5428 net/minecraft/world/gen/UniformIntDistribution
```

Comment:
Can't agree with this name. My first instinct on seeing "base" and spread given this name would be that base is the center and spread is the radius. But it is not.

Reply:
This is more like a "base" and a "bonus" combination

Reply:
It's still a uniform integer distribution though.
I think the name is fine, even though it's Constrained/Clamped

Reply:
Can we get like a shorter name for this? This long name looks weird when I have to use it

Reply:
Any suggestions?